SlicerLite/SlicerLiteLib/SlicerLiteModuleWidget.py

```diff
@@ -0,0 +1,170 @@
+        self.dicom_tags_button_item_delegate = DicomMetadataButtonItemDelegate()
+
+        self.buttons_delegate = [self.delete_button_item_delegate, self.dicom_tags_button_item_delegate]
+
+        # Setup event filter
+        self.filter = DragAndDropEventFilter(slicer.util.mainWindow(), self.load_dicom_directory)
+        slicer.util.mainWindow().installEventFilter(self.filter)
+
+        self.setup_ui()
+
+    def setup_ui(self) -> None:
+        """
+        Create and initialize UI components
+        """
+        self.setup_table_view_layout()
+        self.setup_rendering_layout()
+        self.setup_segmentation_layout()
+        self.layout().addStretch()
+
+    def setup_table_view_layout(self) -> None:
+        """
+        Setup the table view behavior/display
+        """
+        self.item_table_model.setHorizontalHeaderLabels(["Name", "Dicom tags", "Delete"])
+        self.item_table_view.setItemDelegateForColumn(1, self.dicom_tags_button_item_delegate)
+        self.item_table_view.setItemDelegateForColumn(2, self.delete_button_item_delegate)
+        # Hide headers
+        self.item_table_view.horizontalHeader().hide()
+        self.item_table_view.verticalHeader().hide()
+        # Hide table borders
+        self.item_table_view.setFrameStyle(qt.QFrame.NoFrame)
+        self.item_table_view.horizontalHeader().setSectionResizeMode(qt.QHeaderView.Stretch)
+        self.item_table_view.horizontalHeader().resizeSection(1, 32)
+        self.item_table_view.horizontalHeader().resizeSection(2, 32)
+        # Make lines no editabled
+        self.item_table_view.setEditTriggers(qt.QAbstractItemView.NoEditTriggers)
+        # Allow only one selection
+        self.item_table_view.setSelectionMode(qt.QAbstractItemView.SingleSelection)
+
+        self.item_table_view.clicked.connect(self.on_table_view_item_clicked)
+        self.item_table_view.setModel(self.item_table_model)
+
+        self.layout().addWidget(UIUtils.createButton("Load", callback=self.on_click_load_dicom_directory))
+        self.layout().addWidget(qt.QLabel("Loaded volumes:"))
+        self.layout().addWidget(self.item_table_view)
+
+    def setup_rendering_layout(self) -> None:
+        """
+        Get and place shift rendering sliders for volume rendering
+        """
+        self.rendering_module = slicer.util.getNewModuleGui(slicer.modules.volumerendering)
+        self.shift_slider_widget = slicer.util.findChild(self.rendering_module, "PresetOffsetSlider")
+        self.shift_slider_widget.setEnabled(False)
+
+        layout = qt.QHBoxLayout()
+        layout.addWidget(qt.QLabel("Rendering shift:"))
+        layout.addWidget(self.shift_slider_widget)
+
+        self.layout().addLayout(layout)
+
+    def setup_segmentation_layout(self) -> None:
+        """
+        Get and set the segmentation modules and simplify it
+        """
+        segment_editor_module = slicer.modules.segmenteditor.widgetRepresentation()
```

Comment:
We should probably create a new UI instead of stealing the segment editor for this extension